docs/element-definitions.md

```diff
@@ -1915,7 +1915,7 @@ the `inferred` link tells the user that the element was created based on some
 varying assumptions, and the URL to which the link points MAY provide an
 explanation on how and why it was inferred.
 
-## Authentication and Authorization Schemes
+## Authentication and Authorization Scheme
```

Comment:
I think this should be plural, if you read the next sentance we are talking about schemes, not a specific scheme? This is a section describing all of the authentication and authorization schemes.